### ISSUE TITLE
Access Wideners are only remapped if they're specified in fabric.mod.json

### ISSUE THREAD

**Issue by Lassebq** [FabricMC/fabric-loom#1191]
I'm using loom to compile and remap minecraft mixins outside of fabric environment, and fabric.mod.json isn't a requirement there. I'm looking for any way to explicitly remap access wideners. Or if possible, remap any access wideners specified by
```groovy
    loom {
    	accessWidenerPath = file("src/main/resources/modid.accesswidener")
    }
 ```

**Comment by modmuss50** [comment]
Hi, I think the remap jar task would need a way for you to specify what entires within a jar file are acess wideners, and that they need remapping.

**Comment by Lassebq** [comment]
Why not `loom.accessWidenerPath`? Is it because it can point to any AW in a filesystem and not guaranteed to be included with the mod?

**Comment by modmuss50** [comment]
Yes correct, thats the AW that is being applied to the project, and not necessarily one within the jar being remapped.
